### PR TITLE
inspecting sky material

### PR DESCRIPTION
**Pull request by RolandCsibrei**
Hi!
This introduces a dependency on the materials library. I'm not sure if this is a correct approach. Please comment!

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15748/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15748/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15748/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15748/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15748/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15748/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15748/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15748/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15748/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15748/merge/testResults/webgl2playwright/index.html

**Review by deltakosh — APPROVED** [review]

File: packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/materials/skyMaterialPropertyGridComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+import type { SkyMaterial } from "materials/sky/skyMaterial";
+
+import type { PropertyChangedEvent } from "../../../../propertyChangedEvent";
+import { LineContainerComponent } from "shared-ui-components/lines/lineContainerComponent";
+import { CheckBoxLineComponent } from "shared-ui-components/lines/checkBoxLineComponent";
+import { SliderLineComponent } from "shared-ui-components/lines/sliderLineComponent";
+import { CommonMaterialPropertyGridComponent } from "./commonMaterialPropertyGridComponent";
+import type { LockObject } from "shared-ui-components/tabs/propertyGrids/lockObject";
+import type { GlobalState } from "../../../../globalState";
+import { Vector3LineComponent } from "shared-ui-components//lines/vector3LineComponent";
```
Comment (Popov72) [inline packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/materials/skyMaterialPropertyGridComponent.tsx:13]:
```suggestion
import { Vector3LineComponent } from "shared-ui-components/lines/vector3LineComponent";
```

**Pull request merged**